src/services/onboard/ledger-module.ts

```diff
@@ -0,0 +1,349 @@
+import type { DeviceActionState } from '@ledgerhq/device-management-kit'
+import type { SignTransactionDAOutput, SignTypedDataDAOutput, TypedData } from '@ledgerhq/device-signer-kit-ethereum'
+import type { GetAddressCommandResponse } from '@ledgerhq/device-signer-kit-ethereum/api/app-binder/GetAddressCommandTypes.js'
```

Comment:
Can be imported directly from the index on the package. it should not be imported directly with a direct path.

Reply:
It doesn't seem possible as `GetAddressCommandResponse` is not exported from the relevant [barrel file](https://github.com/LedgerHQ/device-sdk-ts/blob/develop/packages/signer/keyring-eth/src/api/index.ts).

Reply:
Hello 👋  In fact it's the wrong type that you are using, it should be `GetAddressDAOutput`, and this one is accessible 👍

Reply:
(in the end they are equal)